hack/grab-profiles.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SERVER_ADDR=""
```

Comment:
per the Google shell style guide, unless these are coming from the environment they shouldn't be capitalized (https://google-styleguide.googlecode.com/svn/trunk/shell.xml?showone=Variable_Names#Variable_Names)